tasks/components/UserMenu.tsx

```diff
@@ -34,14 +33,9 @@ export const UserMenu = ({
 }: PropsWithLanguage<UserMenuTranslation>) => {
   const translation = useTranslation(language, defaultUserMenuTranslations)
   const [isLanguageModalOpen, setLanguageModalOpen] = useState(false)
+  const { user, signOut } = useAuth()
 
-  const router = useRouter()
-  const auth = useAuth(() => router.push({ pathname: '/login', query: { back: true } }))
-
-  const user = auth.user
-  if (!user) return <></>
-
-  const logout = () => auth.logout(() => router.replace('/'))
+  if (!user) return null
 
   // TODO update this when introducing proper auth
   const settingsURL = 'https://auth.helpwave.de/ui/settings'
```

Comment:
Thats part of this issue I guess. Is that static or should we move it to config?

Reply:
fbe88e8a2f1f5a2522563f3cea3da817b9d7cb38